code/game/machinery/nuclear_bomb.dm

```diff
@@ -23,11 +23,22 @@ var/bomb_set
 	var/previous_level = ""
 	var/datum/wires/nuclearbomb/wires = null
 
+	var/eris_ship_bomb = FALSE           // if TRUE (1 in map editor), then Heads will get parts of code for this bomb. Obviously used in map editor. Single mapped bomb supported.
+	var/ship_nuke_code_rotation_part = 1 // Which part of code next spawned Head gets. See "get_next_nuke_code_part()" proc.
```

Comment:
Unused var?

Reply:
used by copytext() in get_next_nuke_code_part() proc below, it guides procedure which digits we need to copy from password. i can add this in comment if needed.

Reply:
As far as i can see `get_next_nuke_code_part()` is using gametickers variable of same name, not that one.